starter-project-web-react/cypress/e2e/blogPost.spec.cy.ts

```diff
@@ -0,0 +1,63 @@
+describe('Blog Post', () => {
+    beforeEach(()=>{  
+      cy.login(Cypress.env('TEST_EMAIL'), Cypress.env('TEST_PASSWORD')) 
+    })
+  
+    it('should post blog ', () => {
+      cy.visit('/blogs')
+
+      const title = "TEST_TITLE new 1"
+      const content = "Test_Content 1"
+      cy.contains('Post Blog').click()
+      cy.get('[name="title"]').clear()
+      cy.get('[name="content"]').clear()
+      cy.get('[name="title"]').type(title)
+      cy.get('[name="content"]').type(content)
+      
+      cy.contains("POST").click()
+      cy.wait(3000)
+      cy.contains(title).should('be.visible')
+      cy.contains(content).should('be.visible')
+
+      
+    })
+
+    it('should show error message for empty content ', () => {
+      cy.visit('/blogs')
+
+      const title = "TEST_TITLE 1 error"
+      const content = " "
+      cy.contains('Post Blog').click()
+      cy.get('[name="title"]').clear()
+      cy.get('[name="content"]').clear()
+      cy.get('[name="title"]').type(title)
+      cy.get('[name="content"]').type(content)
+      
+      cy.contains("POST").click()
+      cy.wait(3000)
+      cy.contains("Title can not be only space").should('be.visible')
+      // cy.contains(content).should('be.visible')
+
+      
+    })
+
+    it('should show error message for empty title', () => {
+      cy.visit('/blogs')
+
+      const title = " "
+      const content = "Test_Content error"
+      cy.contains('Post Blog').click()
+      cy.get('[name="title"]').clear()
+      cy.get('[name="content"]').clear()
+      cy.get('[name="title"]').type(title)
+      cy.get('[name="content"]').type(content)
+      
+      cy.contains("POST").click()
+      cy.wait(3000)
```

Comment:
wait is not recommended, instead, there is a way to intercept api calls and do an action when the api call returns
additionally, the API can be mocked to return something without making a call and not depending on the backend because their database may change later and the test should survive that